silkworm/db/datastore/snapshots/domain_range_latest_query.hpp

```diff
@@ -46,7 +46,7 @@ struct DomainRangeLatestSegmentQuery {
 
     using ResultItem = btree::BTreeIndex::Cursor::value_type;
 
-    auto exec_with_eager_begin(Bytes key_start, Bytes key_end, bool ascending) {
+    auto exec_with_eager_begin(const Bytes& key_start, Bytes key_end, bool ascending) {
```

Comment:
I recommend to add NOLINT here or pack keys into std::pair in order to preserve symmetry.